lib/prawn/embedded_files.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Prawn
+  module EmbeddedFiles
+    # include PDF::Core::EmbeddedFiles
+    # Link to PR (https://github.com/prawnpdf/pdf-core/pull/47)
+
+    def file(src, options = {})
+      path = Pathname.new(src)
+
+      if path.directory?
+        raise ArgumentError, 'Data source can\'t be a directory'
+      elsif path.file?
+        data = path.read
+        options[:name] ||= src
```

Comment:
You could add the options `:modification_date` and `:creation_date` here based on `path.mtime` and `path.ctime`.

Reply:
Sorry, I missed that. Fixed.